### ISSUE TITLE
CA1720 should not warn here

### ISSUE THREAD

**Issue by JohanLarsson** [dotnet/roslyn-analyzers#2731]
```cs
public static void M(int obj) { }
```

**Comment by GammaFour** [comment]
Yeah, I have an issue with this also.  This line throws a warning:

```
        public async Task<IActionResult> PostStampedProposedOrder([FromBody] object @object)
        {

```
If I've gone to the trouble of prefacing the variable name with an @, it means I understand the rule and have explicitly overrriden it.  This error should be suppressed for names beginning with @.

**Comment by mavasani** [comment]
Duplicate of https://github.com/dotnet/roslyn-analyzers/issues/1823
